### PR TITLE
config: auto respond to lgtm

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

ref https://github.com/ti-community-infra/tichi/pull/562

### CODE REVIEW

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -675,6 +681,175 @@ ti-community-autoresponder:
...
 
           If you have any questions about the PR merge process, please refer to [pr process](https://book.prow.tidb.io/#/en/workflows/pr).
+      - regex: "(?i)^/LGTM"
+        message: |
+          Please use GitHub review feature instead of `/lgtm [cancel]` when you want to submit review to the pull request.
+          For how to use GitHub review feature, see also [this document](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) provided by GitHub.
+          For the reason we drop support to the commands, see also [this page](https://book.prow.tidb.io/#/en/plugins/lgtm?id=why-does-lgtm-cancel-take-no-effect-anymore).
+          This reply is being used as a temporary reply during the migration of review process and will be removed on July 1st.
+  - repos:
+    - tikv/community
```
Comment (hi-rustin) [inline prow/config/external_plugins_config.yaml:691]: This is a list, you don't need to copy it so many times. Just add more repos and you're done.

**Review by hi-rustin — COMMENTED** [review]
Thanks!

File: prow/config/external_plugins_config.yaml [hunk 2]
```diff
@@ -675,6 +681,175 @@ ti-community-autoresponder:
...
+  - repos:
+      - chaos-mesh/website-zh
+    auto_responds:
+      - regex: "(?i)^/LGTM"
+        message: |
+          Please use GitHub review feature instead of `/lgtm [cancel]` when you want to submit review to the pull request.
+          For how to use GitHub review feature, see also [this document](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) provided by GitHub.
+          For the reason we drop support to the commands, see also [this page](https://book.prow.tidb.io/#/en/plugins/lgtm?id=why-does-lgtm-cancel-take-no-effect-anymore).
+          This reply is being used as a temporary reply during the migration of review process and will be removed on July 1st.
+
```
Comment (hi-rustin) [inline prow/config/external_plugins_config.yaml:852]:
```suggestion
```

File: prow/config/external_plugins_config.yaml [hunk 3]
```diff
@@ -663,6 +663,12 @@ ti-community-autoresponder:
           It seems you want to merge this PR, I will help you trigger all the tests:
 
           /run-all-tests
+      - regex: "(?i)^/LGTM"
+        message: |
+          Please use GitHub review feature instead of `/lgtm [cancel]` when you want to submit review to the pull request.
+          For how to use GitHub review feature, see also [this document](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) provided by GitHub.
```
Comment (hi-rustin) [inline prow/config/external_plugins_config.yaml:669]:
```suggestion
          For how to use GitHub review feature, see also [this document](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) provided by GitHub.
          
```

File: prow/config/external_plugins_config.yaml (hunk 1 above)
Comment (tisonkun) [reply prow/config/external_plugins_config.yaml:691]: ok

**Comment by tisonkun** [comment]
updated

**Review by hi-rustin — APPROVED** [review]
/ok-to-test

/assign @Mini256 

PTAL~

**Review by Mini256 — APPROVED** [review]

**Pull request merged**